From c4c0564b48d7e37ad7d828fb60e44eee67434f46 Mon Sep 17 00:00:00 2001
From: Alan Tull <r80115@freescale.com>
Date: Thu, 18 Sep 2008 14:22:25 -0500
Subject: [PATCH] ENGR00092528-2: ASoC: Correct WM8903 DRC slope text

Early revisions of the datasheet contain an error, update to reflect the
chip & current datasheet.

commit 3bba971ef0ada4957c1a57d1cdf592da901156d1 from
Wolfson linux-2.6-asoc repo
http://opensource.wolfsonmicro.com/

Signed-off-by: Alan Tull <r80115@freescale.com>
---
 sound/soc/codecs/wm8903.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/codecs/wm8903.c b/sound/soc/codecs/wm8903.c
index 3e4451f..6d3c022 100644
--- a/sound/soc/codecs/wm8903.c
+++ b/sound/soc/codecs/wm8903.c
@@ -591,7 +591,7 @@ static const DECLARE_TLV_DB_SCALE(drc_tlv_max, 1200, 600, 0);
 static const DECLARE_TLV_DB_SCALE(drc_tlv_startup, -300, 50, 0);
 
 static const char *drc_slope_text[] = {
-	"0", "1/16", "1/8", "1/4", "1/2", "1"
+	"1", "1/2", "1/4", "1/8", "1/16", "0"
 };
 
 static const struct soc_enum drc_slope_r0 =
-- 
1.5.4.4

